src/main/java/com/example/javaOjt/controllers/AuthorController.java

```diff
@@ -69,4 +73,18 @@ public ResponseEntity<GetAuthorResponse> getAuthor(
     return ResponseEntity.ok(response);
   }
 
+  /**
+   * Add a new author to the database by the author's name.
+   *
+   * @param requestBody JSON with the key "name". Any other value will be ignored.
+   * @return a ResponseEntity containing the URI to the added author in the headers.
+   */
+  @PostMapping("")
+  public ResponseEntity<GetAuthorResponse> postAuthor(
+      @RequestBody @Validated AuthorRequest requestBody
+  ) {
+    GetAuthorResponse savedAuthor = authorService.postAuthor(requestBody.getName());
+    return ResponseEntity.created(URI.create("/authors/" + savedAuthor.getId())).body(null);
```

Comment:
レスポンスボディの内容指定が漏れたようですね。

```suggestion
    return ResponseEntity.created(URI.create("/authors/" + savedAuthor.getId())).body(savedAuthor);
```

Reply:
あっ、すみません、[そのまま返したくない](https://github.com/Flyover1598/java-ojt/pull/12#discussion_r2166400149)ってのを誤解してしまって、[空で返す](https://github.com/b02902092/java-ojt/issues/3#issue-3099728790)のかな…って思いこんでしまいました。お手数をおかけしてしまい申し訳ありません🙇